src/main/java/com/github/instagram4j/instagram4j/actions/account/AccountAction.java

```diff
@@ -47,4 +44,18 @@
     public CompletableFuture<AccountsUserResponse> currentUser() {
         return new AccountsCurrentUserRequest().execute(client);
     }
+
+    /**
+     * There is a daily limit to the use of setStatus.
+     *
+     * @param text The thread app limit the String to a length to 30 characters, but there is no server verification
+     * @param emoji The thread app prevent you to put anything else that an emoji, but there is no server verification
+     * @param expires_at Must be in second (Example System.currentTimeMillis()/1000+60 for 60 seconds)
+     * @param should_notify Close friend should be notify or not
+     * @param status_type Must be "auto" or "manual"
+     * @return IGResponse
+     */
+    public CompletableFuture<IGResponse> setStatus(String text, String emoji, long expires_at, boolean should_notify, String status_type) {
```

Comment:
Nice documentation.

I would like to see another overloaded method that sets defaults. (Such as text, emoji, and expires_at with default should_notify false and manual status_type I assume)